package.json

```diff
@@ -54,6 +56,10 @@
     "react": ">= 16",
     "react-dom": ">= 16"
   },
+  "exports": {
+    ".": "./node_package/lib/ReactOnRails.js",
+    "./*": "./node_package/lib/*.js"
+  },
```

Comment:
@KhaledEmaraDev can you explain this?

Seems odd to start exporting everything from package.json

Why not add what's necessary to the current exports?

We already have:
https://github.com/shakacode/react_on_rails/blob/master/node_package/src/ReactOnRails.ts#L292-L293